core/src/main/java/nl/nn/adapterframework/core/ProcessState.java

```diff
@@ -15,27 +15,39 @@
 */
 package nl.nn.adapterframework.core;
 
-import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.LinkedHashSet;
 import java.util.Map;
 import java.util.Set;
 
+import org.apache.commons.lang3.StringUtils;
+
+import lombok.Getter;
+import lombok.Setter;
+
 public enum ProcessState {
 
-	AVAILABLE,
-	INPROCESS,
-	DONE,
-	ERROR,
-	HOLD;
+	AVAILABLE("Available"),
+	INPROCESS("InProcess"),
+	DONE("Done"),
+	ERROR("Error"),
+	HOLD("Hold");
+
+	@Getter @Setter
+	private String name;
 
+	private ProcessState(String name) {
+		this.name=name;
+	}
+	
 	public static Set<ProcessState> getMandatoryKnownStates() {
 		Set<ProcessState> knownProcessStates = new LinkedHashSet<>();
 		knownProcessStates.add(AVAILABLE);
 		return knownProcessStates;
 	}
 
 	public static Map<ProcessState, Set<ProcessState>> getTargetProcessStates(Set<ProcessState> knownProcessStates) {
-		Map<ProcessState, Set<ProcessState>> targetProcessStates = new HashMap<>();
+		Map<ProcessState, Set<ProcessState>> targetProcessStates = new LinkedHashMap<>();
```

Comment:
Turning this into a LinkedHashMap suggests that the insertion order is significant. Is that the case? Are there iterations over this map that should be in the right order?